post_it_backend/settings.py

```diff
@@ -23,7 +23,7 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = env('SECRET_KEY')
+SECRET_KEY = 'django-insecure-!l5_lbaz2+e6)&haqh@caqe_ixlr5oukfxuufqw+h3=5haztej'
```

Comment:
Why are you using the key direct here?

Reply:
To give access to anyone testing. I am not sure if this key can be generated locally since it's generated on `python manage.py startapp` command given the project app folder is already set.

Reply:
No need, everyone can have their own secret key, so please revert back